services/web/server/src/simcore_service_webserver/exporter/_handlers.py

```diff
@@ -15,7 +15,7 @@
 from .._meta import API_VTAG
 from ..constants import RQ_PRODUCT_KEY
 from ..login.decorators import login_required
-from ..projects.projects_service import create_user_notification_cb
+from ..projects._projects_service import create_user_notification_cb
```

Comment:
why is this module still called a handler?
handler = controller if I got this CSR right (even if I don't like the name).. I find handler more precise. but I find mixing everything worse.

Reply:
Unfortunately, not all domains have been updated to the new naming convention yet. For now, we’ll have to tolerate some inconsistencies across the webserver until the full migration is complete. @bisgaard-itis and I are actively working on getting everything aligned and consistent as soon as possible.